assets/js/vue/components/search/Facet.vue

```diff
@@ -132,5 +139,9 @@
 </script>
 
 <style scoped>
-
-</style>
\ No newline at end of file
+    .scrollable-facet {
+        height: auto;
+        max-height: 20rem;
```

Comment:
Maybe this should be em, but then again, maybe I don't really undestand these unites.

Reply:
Im not sure either